agent/engine/task_manager.go

```diff
@@ -129,9 +133,15 @@ func (mtask *managedTask) overseeTask() {
 			// able to move some containers along.
 			llog.Debug("Task not steady state or terminal; progressing it")
 
-			// TODO:
-			// 1) Add platform resources setup and failure handling
-			// 2) Add new task resources provisioned state ?
+			// TODO: Add new resource provisioned state ?
+			if mtask.engine.cfg.TaskCPUMemLimit {
+				err := mtask.resource.Setup(mtask.Task)
+				if err != nil {
+					seelog.Criticalf("Unable to setup platform resources for task %s: %v", mtask.Task.Arn, err)
+					mtask.SetDesiredStatus(api.TaskStopped)
+					mtask.engine.emitTaskEvent(mtask.Task, taskUnableToCreatePlatformResources)
+				}
+			}
```

Comment:
Please add a log line here indicating that the cgroup hierarchy has been established for the task.

Reply:
Can I add it after #989 is merged and we update the transition dependencies as this log line would now appear on every task transition ?

Reply:
Yeah, that's fine. You can add a TODO for the time being so that we won't forget it.

Reply:
Done! 
Thanks